content/en/blog/_posts/2022-05-03-kubernetes-release-1.24.md

```diff
@@ -0,0 +1,229 @@
+---
+layout: blog
+title: "Kubernetes 1.24: Stargazer"
+date: 2022-05-03
+slug: kubernetes-1-24-release-announcement
+---
+
+# Kubernetes 1.24
+
+**Authors**: [Kubernetes 1.24 Release Team](https://github.com/kubernetes/sig-release/blob/master/releases/release-1.24/release-team.md)
+
+We are excited to announce the release of Kubernetes 1.24, the first release of 2022!
+
+This release consists of 46 enhancements: fifteen enhancements have graduated to stable, 
+fifteen enhancements are moving to beta, and thirteen enhancements are entering alpha. 
+Also, two features have been deprecated, and the [dockershim has been removed](/dockershim).     
```

Comment:
15+14+13+2 and... +1 for dockershim to make 46?

Reply:
Those numbers come from the spreadsheet, but I think it’s wrong as it doesn’t add up, and dynamic kubelet config got removed too.

Looking at the spreadsheet in more detail, Dockershim is miscategorised as “stable” so it’s counting towards that 15. I think a more accurate statement is:

```suggestion
This release consists of 46 enhancements: fourteen enhancements have graduated to stable, 
fifteen enhancements are moving to beta, and thirteen enhancements are entering alpha. 
Also, two features have been deprecated and two features have been removed.     
```

14+15+13+2+2=46